classes/Component.js

```diff
@@ -1,19 +1,24 @@
 const fs = require('fs-extra');
+const path = require('path');
 const replace = require('replace');
 
 const templates = require('../templates/component');
 
+const { getConfig } = require('../config');
 const { capitalize } = require('../utils');
 
+const projectConfig = getConfig();
 class Component {
   constructor(component, options) {
+    const workingDir = process.cwd();
+    const componentsDir = path.normalize(`${workingDir}/${projectConfig.root}/components`);
+    const rootDir = path.normalize(`${workingDir}/${projectConfig.root}`);
+
     this.component = capitalize(component);
     this.options = options;
-    this.basePath = fs.existsSync('./components')
-      ? './components'
-      : '';
-    this.directoryPath = `${this.basePath}/${this.component}`;
-    this.componentPath = `${this.directoryPath}/${this.component}`;
+    this.basePath = fs.existsSync(componentsDir) ? componentsDir : rootDir;
```

Comment:
I'm think that if you go deeper intro folder instead of running this command in root of project you will create a new path and create component there, did you check it?

Reply:
No, I didn't check it, but it's ok. If you don't have set `root` option in `.rdc` file and will go to the `src` folder manually, it's ok to create a component there, isn't it?